reports/RJ722-II.md

```diff
@@ -0,0 +1,370 @@
+---
+layout: report
+permalink: /GSoC/2018/StatusReport/RahulJha
```

Comment:
https://deploy-preview-622--coala-projects.netlify.com/GSoC/2018/StatusReport/RahulJha not working

Reply:
There was a problem with parsing the ":" in the commit messages. Resolved!